Description: Add support for m68k.
 This patch adds support for m68k by defining the correct
 macro to determine the instruction pointer from ucontext.
 .
Author: John Paul Adrian Glaubitz <glaubitz@physik.fu-berlin.de>
Bug-Debian: https://bugs.debian.org/865726
Forwarded: http://gmt.soest.hawaii.edu/issues/1122
Applied-Upstream: http://gmt.soest.hawaii.edu/projects/gmt/repository/revisions/18449
Last-Update: 2017-06-24

--- a/src/common_sighandler.c
+++ b/src/common_sighandler.c
@@ -83,6 +83,8 @@ void backtrace_symbols_fd(void *const *b
 #  define UC_IP(uc) ((void *) (uc)->uc_mcontext.arm_pc)
 # elif defined( __hppa__)
 #  define UC_IP(uc) ((void *) (uc)->uc_mcontext.sc_iaoq[0])
+# elif defined(__m68k__)
+#  define UC_IP(uc) ((void *) (uc)->uc_mcontext.gregs[R_PC])
 # elif defined(__s390__)
 #  define UC_IP(uc) ((void *) (uc)->uc_mcontext.psw.addr)
 # elif defined(__sparc__)
